halo2-wasm/js/kzg/index.ts

```diff
@@ -34,11 +42,33 @@ const convertBase64ToUint8Arr = (b64str: string) => {
     buf[i] = ch.charCodeAt(0);
   });
   return buf;
-}
+};
 
 export const getKzgParams = async (k: number): Promise<Uint8Array> => {
+  const home = os.homedir();
```

Comment:
I think just like `fetchAndConvertToUint8Array` we need to first check if this is running in Node.js or browser, and only use/import `os`/`path`/`https`  if it's node. And maybe we could save to localStorage if it's from browser?